src/components/global/Playground/playground.css

```diff
@@ -181,19 +181,8 @@
 
 /* The code block example that users can copy to clipboard */
 .playground__code-block {
-  /*
-   * Height is set to zero instead of display: none so that when copying the contents
-   * of the code block to use in Stackblitz, the individual line breaks are preserved.
-   * Otherwise when using display: none, the result of el.outerText will not include
-   * the individual "\n" line breaks and the code snippet will be one line.
-   */
-  height: 0px;
-  overflow: hidden;
-}
-
-.playground__code-block--expanded {
-  height: initial;
   margin-top: 16px;
+  overflow: hidden;
```

Comment:
Is `overflow: hidden` still necessary? Originally it was added to hide the container until it was expanded. Since the container is now expanded by default (with no option to collapse), I don't believe we need to worry about overflow.

Reply:
Good call!